input-explainer.md

```diff
@@ -226,8 +226,6 @@ For `tracked-pointer` input sources, it is often appropriate for the application
 
 
 #### Choosing renderable models
-The majority of `tracked-pointer` input sources will have a non-null `gamepad` attribute on the `XRInputSource` object. The `Gamepad`'s `id` is used to determine what should be rendered if the app intends to visualize the input source itself, rather than an alternative virtual object. (See the section on [Button and Axis State](#button-and-axis-state) for more details.)
```

Comment:
Ewe. This was just flat out wrong, wasn't it? (The use of the Gamepad.id for models)

Reply:
yeeeeeep.